tests/unit/plugins/test_qmake_plugin.py

```diff
@@ -0,0 +1,106 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2020-2022 Canonical Ltd.
```

Comment:
```suggestion
# Copyright 2023 Canonical Ltd.
```

Feel free to add yourself or your org to the copyright block too (I'm guessing the copyright is partially Canonical's since I presume that more than the copyright block was copied from elsewhere - correct as needed :-) ) And same note about the copyright owners on the plugin file too.

Reply:
That is correct :) I followed a commit for the cmake plugin and converted to qmake logic. I will add myself.